### ISSUE TITLE
Nvim-R inappropriately evaluates active bindings

### ISSUE THREAD

**Issue by klmr** [jalvesaq/Nvim-R#686]
I have the following active binding defined for convenience inside my `~/.Rprofile`:

```r
if (interactive()) {
    makeActiveBinding("qq", \() quit(save = "no"), .GlobalEnv)
}
```

Unfortunately, in combination with the above, the following sequence of steps causes Nvim-R to quit the R session:

1. Open a new R file with nvim: `nvim x.r`
2. Enter an expression into the file, e.g. `1 + 2`
3. Start Nvim-R (<kbd>\\</kbd><kbd>r</kbd><kbd>f</kbd>)
4. Execute the expression (<kbd>\\</kbd><kbd>d</kbd>)
5. Go into insert mode (<kbd>i</kbd>)

Result: the R session quits.

I’m guessing that Nvim-R is trying to list the symbols in the global environment for OmniComplete or something, and is accidentally evaluating the active binding in the process. Indeed, other active bindings are also evaluated, e.g. defining the following:

```r
makeActiveBinding("x", \() message("foo"), .GlobalEnv)
```

… causes Nvim-R to print `foo` on the R terminal when going into insert mode after sending an expression to R.

To fix this, something like the following might be used:

```r
if (bindingIsActive(name, env)) {
    # special handling for active bindings
} else {
    # existing logic, e.g.:
    get(name, env)
}
```

In combination with the following, if the value (= function) of the active binding is required:

https://github.com/klmr/box/blob/beaa376ce71afb549931c9027c9dc8e63a400c44/R/env.r#L226-L232

**Comment by jalvesaq** [comment]
Thanks for reporting the issue and investigating its origin. The last commit removed active bindings from the list of objects for both omni completion and the Object Browser. It would be better if we could find a way of listing them without getting them evaluated.

**Comment by klmr** [comment]
R defines the internal function [`R_ActiveBindingFunction`](https://github.com/wch/r-source/blob/a27f10446fae52ef60c1a9aae07947cfd5c6d3db/src/include/Rinternals.h#L673) for this purpose. If you prefer not to access internals, you’d have to use an R function call to `activeBindingFunction` as shown in the linked code in my comment above. However, that function is only available since R 4.0.0.

**Comment by jalvesaq** [comment]
Thanks again! Now, active bindings are listed in the Object Browser.

`nvimcom.c` was already using `Rinternal.h`.
